invoice-maker.typ

```diff
@@ -1,4 +1,5 @@
 #let nbh = "‑"
```

Comment:
You hardcoded the currency 😅. Can you please make it changeable with default €?

Reply:
you hardcoded it first ;)
should be better now, not sure it makes sense to link the currency to the language, made it overridable in the invoice data, good enough for my needs. default is back to € again.